packages/wrangler/src/pages/dev.ts

```diff
@@ -1269,6 +1270,16 @@ function getBindingsFromArgs(args: typeof pagesDevCommand.args): Partial<
 					return;
 				}
 
+				const bucketName = ref || binding.toString();
+
+				if (!isValidR2BucketName(bucketName)) {
+					logger.error(
+						`The bucket name "${bucketName}" is invalid. ` +
+							"Bucket names must begin and end with an alphanumeric, only contain letters (a-z), numbers (0-9), and hyphens (-), and be between 3 and 63 characters long."
+					);
+					return;
```

Comment:
Why the quiet exit here?
If you throw a UserError then the process will exit with a non-zero exit code.

Reply:
Because this is similar to line 1270 and I don't any `UserError` here.